### ISSUE TITLE
After 3.3.4 to 3.4.1 upgrade `service postgresql start` fails to start postgres server in the container

### ISSUE THREAD

**Issue by ishustava** [concourse/concourse#1566]
We have a docker image, which installs postgres (`v.9.6`) and a task which starts postgres server in the container locally by executing:
```
service postgresql start
```

This command stopped working in concourse version `3.4.1`. The service reports that it has started successfully, however, `service postgresql status` shows that the server is down.

Concourse versions: `3.3.4` to `3.4.1`
Deployment: BOSH
Infrastructure: GCP

### What fixed it
We have switched to using `pg_ctl` utility included with the postgres installation, and it has fixed it for us.

### Why is it a bug?
Upgrade broke previously functional code. We would prefer to use `service` utility which is the best practice.

**Comment by jtarchie** [comment]
Are you're using a bosh release? There is nothing `service` related with the bosh release, it uses `monit` exclusively.

**Comment by kellygerritz** [comment]
@jtarchie I think there's been some confusion. This is not being done within the bosh vm, we're referring to the container in which our concourse task is being executed. We're just providing the information that we deployed our concourse with BOSH.

**Comment by jtarchie** [comment]
I don't see anything here that points it to being concourse, though. Do you have an error messaging?

**Comment by kellygerritz** [comment]
I'm actually having little to no luck finding error messaging, the postgres log file is empty and the `start postgresql start` command returns 0. I've run the same command on docker with our image and that works just fine. We get `[ ok ] Starting PostgreSQL 9.6 database server: main` when we run the start command, but when i run `ps aux | grep -i postgres` all that's returned is the grep command. As far as I can see the service start seems to be a noop not a failure. I don't understand why concourse would cause this either, but the only different variable we can see is that we upgraded the concourse version.

**Comment by qibobo** [comment]
@ishustava  how do you use pg_ctl? I login the container and input pg_ctl, it shows error:pg_ctl: cannot be run as root
Please log in (using, e.g., "su") as the (unprivileged) .

**Comment by qibobo** [comment]
@ishustava  Thank you. It works in my container.

**Comment by vito** [comment]
Could this be the same as https://github.com/concourse/concourse/issues/1578?

**Comment by ishustava** [comment]
@vito Yes, that is exactly the same problem. Which one of these issues would you prefer to keep open?

**Comment by vito** [comment]
Let's close this one since the other has more info about the root cause.
